py_irt/cli.py

```diff
@@ -141,7 +141,9 @@ def evaluate(
         )
 
     observation_subjects = [entry["subject_id"] for entry in subject_item_pairs]
-    observation_items = [entry["item_id"] for entry in subject_item_pairs]
+    # since all subjects use same set of items for predictions (as visible from 
+    # squad.jsonlines), getting all items from one subject
+    observation_items = subject_item_pairs[0]["responses"].keys()
```

Comment:
See my comment in issue #19. Your change requires predictions for all items for a given subject, whereas the prior implementation is a bit more general. For your case, you'd just want to convert your subject-responses jsonlines file to a subject-item jsonlines file.

Reply:
FYI I've updated the README to specify the data format for evaluation to help with this.